<?xml version="1.0"?>
<launch>
    <include file="$(find second_assignment)/launch/sim_w1.launch" />
    <param name="des_pos_x" value= "0.0" />
    <param name="des_pos_y" value= "1.0" />
    <param name="window_size" value= "10" />
    <node pkg="second_assignment" type="wall_follow_service.py" name="wall_follower" />
    <node pkg="second_assignment" type="go_to_point_service.py" name="go_to_point"  />
    <node pkg="second_assignment" type="node_action_client.py" name="action_client" launch-prefix="xterm -e" />
    <node pkg="second_assignment" type="bug_as.py" name="bug_action_service"/>
    <node pkg="second_assignment" type="subscriber_pos_vel.py" name="subscriber_pos_vel" launch-prefix="xterm -e" />
    <node pkg="second_assignment" type="last_target_service.py" name="last_target_service" launch-prefix="xterm -e" />
</launch>